WD-Firmware/board/nexys_a7_swerv_eh1/nexys_a7_swerv_eh1_swervolf.launch

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
This file should be under the new folder (according the tree in README) - /board/nexys_a7_swervolf_eh1/..

Reply:
correct. for now we will keep it here until next round where we will remove the old fpga